datasetIngestor/getLocalFileList.go

```diff
@@ -146,19 +147,20 @@ func GetLocalFileList(sourceFolder string, filelistingPath string, symlinkCallba
 			if runtime.GOOS == windows {
 				modpath = strings.Replace(path, "\\", "/", -1)
```

Comment:
Old code, but filepath.ToSlash would be clearer

Reply:
this code snippet converts from backslash to forward slash when running Windows. filepath.ToSlash would only do forward slash to backslash under Windows.

Reply:
> [ToSlash](https://pkg.go.dev/path/filepath#ToSlash) returns the result of replacing each separator character in path with a slash ('/') character

Reply:
This doesn't really matter since it only runs on windows anyways